descope/management/sso_settings.py

```diff
@@ -0,0 +1,156 @@
+from typing import List
+
+from descope.auth import Auth
+from descope.management.common import MgmtV1
+
+
+class RoleMapping:
+    def __init__(self, groups: List[str], role_name: str):
+        self.groups = groups
+        self.role_name = role_name
+
+
+class SSOSettings:
+    _auth: Auth
+
+    def __init__(self, auth: Auth):
+        self._auth = auth
+
+    def configure(
```

Comment:
We are missing here all the attributes mapping that should be configure as well to make things work. these attributes are optionals (but for making the role mappings to work the customer must map his Group attribute to our Group attribute.
<img width="1161" alt="image" src="https://user-images.githubusercontent.com/105154237/203432064-a7da6a17-0ad2-4f2a-b2d7-e85080f7bb59.png">

Reply:
It has a dedicated function - see `descope_client.mgmt.sso.map_roles(...)`

Reply:
Its not the same. your function handle the role mapping and not the attributes mapping.

Reply:
@itaihanski did you add this missing functionality ? I cant find it and its important as otherwise all role mappings will not work (and also getting the other attributes values.

Reply:
I see. It will be added after it is supported by the service.